.circleci/config.yml

```diff
@@ -19,12 +19,13 @@ workflows:
           requires:
             - react-tools/check
           image: pennlabs/website
-          tag: '${CIRCLE_SHA1},latest'
+          tag: '${CIRCLE_SHA1},${CIRCLE_BUILD_NUM},latest'
```

Comment:
We probably don't need to tag on SHA any more

Reply:
Agreed. SHA isn't meanginful since it doesn't capture everything about the image.